pydal/helpers/classes.py

```diff
@@ -552,7 +553,7 @@ def close_connection(self):
             self.db.executesql(
                 "DELETE FROM web2py_filesystem WHERE path='%s'" % self.filename
             )
-            query = "INSERT INTO web2py_filesystem(path,content) VALUES ('%s','%s')"
+            query = "INSERT INTO web2py_filesystem(path,content) VALUES (%s, %s)"
```

Comment:
I think this fix should be:
```
query = "INSERT INTO web2py_filesystem(path,content) VALUES ('?', '?')"
```

Reply:
I will take this PR but you are right we should replace all '%s' with '?'. It also requires some refactoring. I will take another PR with that. :-)